### PR TITLE
Refine dynamic networking doc phrasing and formating

### PR DESCRIPTION
**Pull request by ormergi**
Following #729, this PR fix's misplaced statements and refine phrasing and formatting.

### CODE REVIEW

File: docs/operations/hotplug_interfaces.md [hunk 1]
```diff
@@ -77,13 +77,9 @@ spec:
...
     }'
 ```
-
-Please refer to the
-[multus documentation](https://github.com/k8snetworkplumbingwg/multus-cni/blob/master/docs/how-to-use.md#create-network-attachment-definition)
-for more info.
-
-> *NOTE*
-> `virtctl` `addinterface` and `removeinterface` commands are no longer available, hotplug/unplug interfaces is done by editing the VM spec template.
+Please refer to
```
[the remark this answers is not in the document]
Comment (ormergi) [reply docs/operations/hotplug_interfaces.md]: Done all over

File: docs/operations/hotplug_interfaces.md [hunk 2]
```diff
@@ -233,7 +231,9 @@ template:
 
 At this point the subject interface should be detached from the guest but exist in the pod.
 
-### Migrate the VM
+>**Note**: Existing VMs from version v0.59.0 and below does not support hot-unplug interfaces.
```
[the remark this answers is not in the document]
Comment (ormergi) [reply docs/operations/hotplug_interfaces.md]: Done, all over

**Comment by ormergi** [comment]
> Two minor grammatical refinements, otherwise, great clean-up!

Thank you @cwilkers , please take another look

**Comment by cwilkers** [comment]
Looks great!
/lgtm
/approve

**Pull request merged**